healvis/observatory.py

```diff
@@ -230,20 +240,24 @@ def set_beam(self, beam="uniform", freq_interp_kind="linear", **kwargs):
         Set the beam of the array.
 
         Args:
-            beam : str
-                Input to PowerBeam or AnalyticBeam. If beam is
-                a viable input to AnalyticBeam, then instantiates
-                an AnalyticBeam, otherwise assumes beam is a filepath
-                to a beamfits and instantiates a PowerBeam.
+            beam : str, or list of beam objects
+                str: If it is a viable input to AnalyticBeam, 
+	            then instantiates an AnalyticBeam, otherwise assumes beam is 
+                    a filepath to a beamfits and instantiates a PowerBeam.
+                list: List of beam objects. This allows for external beams to be 
```

Comment:
I'm not a fan of the term "external" beam here. I think what you mean are the beam classes defined in pyuvdata and pyuvsim?

Since they're treated as antenna beams, it might be better to refer to them as such.

Reply:
I'll change to antenna beams. We've been using pyuvsim AnalyticBeam but mostly the new PolyBeam types in hera_sim  that are derived from AnalyticBeam.